src/base.rs

```diff
@@ -105,7 +157,7 @@ impl GeneralTaxonomy {
     /// the can be used to speed up some operations by avoiding a string lookup.
     #[allow(clippy::wrong_self_convention)]
     #[inline]
-    pub fn from_internal_id(&self, tax_id: IntTaxID) -> Result<&str> {
+    pub fn from_internal_id(&self, tax_id: IntTaxId) -> Result<&str> {
         // TODO: index so we'll get an Err
```

Comment:
is that still a TODO?

Reply:
Yeah, although this message is bad. We should be returning an `Err` if tax_id is out-of-range instead of `panic`ing.